src/server/pps/cmds/cmds.go

```diff
@@ -983,12 +998,13 @@ All jobs created by a pipeline will create commits in the pipeline's output repo
 				return err
 			}
 			defer client.Close()
-			if err := client.StartPipeline(args[0]); err != nil {
-				return errors.Wrap(err, "error from StartPipeline")
+			if err := client.StartProjectPipeline(pfs.DefaultProjectName, args[0]); err != nil {
+				return errors.Wrap(err, "error from StartProjectPipeline")
 			}
 			return nil
 		}),
 	}
+	startPipeline.Flags().StringVar(&project, "project", "", "Project containing pipeline.")
```

Comment:
Should this default to `pfs.DefaultProjectName`

Reply:
I think that it should default to the contextual project if set, otherwise to the default project (which my current code does not handle).